score-server/src/main/java/bio/overture/score/server/config/ProfileConfig.java

```diff
@@ -0,0 +1,24 @@
+package bio.overture.score.server.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+import java.util.*;
+
+@Configuration
+public class ProfileConfig {
+
+  @Autowired
+  Environment environment;
+
+  @Bean
+  String activeStorageProfile() {
+    Map<String, String> storageProfiles = Map.of("collaboratory", "s3", "azure", "az", "test", "test");
```

Comment:
- `test` is here twice.
- Can we return a known enum value instead of an arbitrary String? There are really only 2 values we should be returning `S3` and `AZURE`. We can define an enum in score-core since this is a dependency used by the server and client, then we can map each of the server profiles to the corresponding enum value.